docs/sources/reference/components/otelcol/otelcol.connector.spanmetrics.md

```diff
@@ -16,24 +16,27 @@ aggregates Request, Error and Duration (R.E.D) OpenTelemetry metrics from the sp
   view call counts just on `service.name` and `span.name`.
 
   Requests are tracked using a `calls` metric with a `status.code` datapoint attribute set to `Ok`:
+
   ```
   calls { service.name="shipping", span.name="get_shipping/{shippingId}", span.kind="SERVER", status.code="Ok" }
   ```
 
 - **Error** counts are computed from the number of spans with an `Error` status code.
 
-    Errors are tracked using a `calls` metric with a `status.code` datapoint attribute set to `Error`:
-    ```
-    calls { service.name="shipping", span.name="get_shipping/{shippingId}, span.kind="SERVER", status.code="Error" }
-    ```
+  Errors are tracked using a `calls` metric with a `status.code` datapoint attribute set to `Error`:
+
+  ```
+  calls { service.name="shipping", span.name="get_shipping/{shippingId}, span.kind="SERVER", status.code="Error" }
+  ```
 
 - **Duration** is computed from the difference between the span start and end times and inserted
-    into the relevant duration histogram time bucket for each unique set dimensions.
+  into the relevant duration histogram time bucket for each unique set dimensions.
 
-    Span durations are tracked using a `duration` histogram metric:
-    ```
-    duration { service.name="shipping", span.name="get_shipping/{shippingId}", span.kind="SERVER", status.code="Ok" }
-    ```
+  Span durations are tracked using a `duration` histogram metric:
+
+  ```
+  duration { service.name="shipping", span.name="get_shipping/{shippingId}", span.kind="SERVER", status.code="Ok" }
+  ```
 
 > **NOTE**: `otelcol.connector.spanmetrics` is a wrapper over the upstream
 > OpenTelemetry Collector `spanmetrics` connector. Bug reports or feature requests
```

Comment:
Same comment as in https://github.com/grafana/alloy/pull/2012#discussion_r1826095047 to wrap the note in a Note adminiton